lib/rules/syntaxes/slot-attribute.js

```diff
@@ -123,9 +126,12 @@ module.exports = {
     function reportSlot(slotAttr) {
       const componentName = slotAttr.parent.parent.rawName
       if (
-        ignore.has(componentName) ||
-        ignore.has(casing.pascalCase(componentName)) ||
-        ignore.has(casing.kebabCase(componentName))
+        ignorePatterns.some(
+          (pattern) =>
+            pattern.test(componentName) ||
+            pattern.test(casing.pascalCase(componentName)) ||
+            pattern.test(casing.kebabCase(componentName))
```

Comment:
Please store the differently cased component names in variables (just below `componentName` itself), so they don't have to be computed in every iteration.